vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -16127,6 +16189,7 @@ quantitykind:SpectralLuminousEfficiency
   qudt:systemDerivedQuantityKind qudt:SOQ_SI ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Spectral Luminous Efficiency"@en ;
+  skos:broader quantitykind:LuminousEfficacy ;
```

Comment:
Efficiency would be a dimensionless percentage; efficacy is, for example lm/W which is not dimensionless, so one cannot be skos:broader than the other.

Reply:
I never made any changes for LuminousEfficacy as far as I can tell, and in my spreadsheet I have a value for DV but the row is red so nothing there is valid. I did take from the plain text description this: "the ratio of luminous flux (in lumens) to power (usually measured in watts)" and tried to figure out the DV from there, but there was no such DV in the vocabulary so I mentioned it to you.

Reply:
Need to fix LM-PER-W in units:
Remove ref to quantitykind:SpectralLuminousEfficiency
Make DV: L-2, I1, M-1, T3

, and make quantitykind:SpectralLuminousEfficiency dimensionless

Reply:
Did you know that we have the following (nonsensical) dimension vector: A0E0L-2I1M-1H0T3D1

I believe that this 'should be': A0E0L-2I1M-1H0T3D0, which is the DV we need for LM-PER-W. The above is not used anywhere I can see.

Reply:
Added new dimension vector and associated it with LM-PER-W. Made other changes.